docs/events/eboard-meetings.rst

```diff
@@ -0,0 +1,49 @@
+########
+Eboard Meetings
+########
+
+This page documents RITlug eboard meetings.
+Topics include room booking, preparation, and running of the meeting.
+
+
+*****************
+Meeting checklist
+*****************
+
+Pre-meeting
+===========
+
+The President will triage the `RITlug operations board <https://github.com/orgs/RITlug/projects/1>`__, looking for tasks that require attention.
+The label **review at meeting** should be assigned to tasks that need explicit review by eboard.
+
+Meeting
+=======
+
+#. Designate a notetaker, typically the `secretary <../roles/secretary>`__, to record discussion points and action items for each task.
+   The notetaker should record their notes in the comments section of the task.
```

Comment:
"...comments section of *each* task." 
"At the end of each eboard meeting, every task should have a clear set of action items to be worked on."

Reply:
+1 to proposed addition here.